### DISCUSSION TITLE
How to use drawio within a team

### DISCUSSION THREAD

**Discussion by gdover** [jgraph/drawio#4776] [General]
We're a team of 10 and want to use drawio as our primary diagraming tool.  What's the best way to use drawio for a team of 10?  Is there a portal where I can setup all users and enforce policy on where to save drawings?  Office 365 is our platform for communication and documents.

**Comment by davidjgraph** [comment]
Use app.diagrams.net and connect it to OneDrive?

**Reply by gdover** [reply]
Thanks David...
